src/consts/eip712domains.ts

```diff
@@ -2,14 +2,14 @@ import { VAULT_CONTRACTS_V4_2_ANDUP } from '../data'
 import { EIP3009TokensInterface } from './interfaces.consts'
 
 export const EIP3009Tokens: EIP3009TokensInterface = {
-	'1': {
-		'0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48': {
-			chainId: '1',
-			verifyingContract: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
-			name: 'USD Coin',
-			version: '2',
-		},
-	},
+	// '1': {
+	// 	'0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48': {
+	// 		chainId: '1',
+	// 		verifyingContract: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+	// 		name: 'USD Coin',
+	// 		version: '2',
+	// 	},
+	// },
```

Comment:
thought: prob best place to do this would be on our UI/API, but no integrator one actually uses gasless transfers with our sdk so this is perfectly fine